backend/resources/user_resources.py

```diff
@@ -0,0 +1,64 @@
+from flask_restful import Resource, reqparse
+from flask_jwt import jwt_required
+
+from models.user import UserModel
+
+# RESTFUL:
+class User(Resource):
+
+    @jwt_required()
+    def get(self, uid):
+        user = UserModel.find_by_uid(uid)
+        if user:
+            # no need to use jsonify as flask_restful does that for us
+            return user.json()
+        else:
+            return {'message': 'User does not exist'}, 404  # not found
+
+    def post(self, uid):
```

Comment:
yes, when the API starts to work we can always add it. for now we don't have to worry making all API perfect. I left post and put out for auth so to make it easier to test. I think we can add the tag later on.